src/autoscaler/eventgenerator/README.md

```diff
@@ -14,7 +14,7 @@ Data Aggregator is one of the components of CF `app-autoscaler`.
 1. change directory to `app-autoscaler`
 1. run `source .envrc`
 1. pull the submodules : `git submodule update --init --recursive`
-1. change directory to `src/eventgenerator`
+1. change directory to `src/autoscaler/eventgenerator`
```

Comment:
This seems wrong.  It makes sense from the autoscaler/README.md but if you are reading this README, you are already in the right directory.